src/fund_my_watcard/main.py

```diff
@@ -73,10 +80,12 @@ def main():
             report_error(CAN_NOT_FIND_CONFIG_FILE)
 
     if args.version:
-        report_message("v{}".format(VERSION))
+        from .util import PRINT_PREFIX
+
+        print(PRINT_PREFIX + " v{}".format(VERSION))
```

Comment:
Here is why travis not pass. This is a hacky way, we need to find out a better way.